docs/_src/tutorials/tutorials/2.md

```diff
@@ -92,6 +92,40 @@ reader.save(directory="my_model")
 new_reader = FARMReader(model_name_or_path="my_model")
 ```
 
+## Distill your model
+In this case, we have used "distilbert-base-uncased" as our base model. This model was trained using a process called distillation. In this process, a bigger model is trained first and is used to train a smaller model which increases its accuracy. This is why "distilbert-base-uncased" can achieve quite competitive performance while being very small.
+
+Sometimes, however, you can't use an already distilled model and have to distil it yourself. For this case, haystack has implemented a [distillation approach](https://arxiv.org/pdf/1909.10351.pdf).
+
+### Augmenting your training data
+To get the most out of model distillation, we recommend increasing the size of your training data by using data augmentation. You can do this by running the [`augment_squad.py` script](https://github.com/deepset-ai/haystack/blob/master/haystack/utils/augment_squad.py):
+
+
+```python
+# Downloading script
+!wget https://raw.githubusercontent.com/deepset-ai/haystack/master/haystack/utils/augment_squad.py
+# Just replace dataset.json with the name of your dataset and adjust the output path
+!python augment_squad.py --squad_path dataset.json --output_path augmented_dataset.json --multiplication_factor 2
+```
+
+In this case, we use a multiplication factor of 2 to keep this example lightweight. Usually you would use a factor like 20 depending on the size of your training data.
+
+### Running distillation
+Distillation in haystack is done in two steps: First, you run intermediate layer distillation on the augmented dataset to ensure the two models behave similarly. After that, you run the prediction layer distillation on the non-augmented dataset to optimize the model for your specific task.
+
+If you want, you can leave out the intermediate layer distillation step and only run the prediction layer distillation. This way you also do not need to perform data augmentation. However, this will make the model significantly less accurate.
+
+
+```python
+teacher = FARMReader(model_name_or_path="my_model", use_gpu=True)
```

Comment:
I think we should add an example model here in a comment. Something like:
```
# a fine-tuned model e.g. "deepset/​bert-​base-​uncased-​squad2"
```

Reply:
I have added the example.